com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDTarget.cs

```diff
@@ -872,7 +874,7 @@ static class CoreKeywords
             { CoreKeywordDescriptors.Shadow },
             { CoreKeywordDescriptors.ScreenSpaceShadow },
             { CoreKeywordDescriptors.Decals },
-            { CoreKeywordDescriptors.LightList, new FieldCondition(Fields.SurfaceOpaque, true) },
+            { CoreKeywordDescriptors.LightList },
```

Comment:
this is not true, TransparentBackface for example still a define as it can be only cluster

Reply:
fixed